kedro/pipeline/node.py

```diff
@@ -550,9 +537,11 @@ def _validate_inputs(self, func, inputs):
                 func_args = inspect.signature(
                     func, follow_wrapped=False
                 ).parameters.keys()
+                func_name = _get_readable_func_name(func)
+
                 raise TypeError(
-                    "Inputs of function expected {}, but got {}".format(
-                        str(list(func_args)), str(inputs)
+                    "Inputs of '{}' function expected {}, but got {}".format(
+                        func_name, str(list(func_args)), str(inputs)
```

Comment:
Can f-string this now.

Reply:
Awesome!!  Kedro is the only place I have used anything else in years.